fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -90,7 +92,7 @@
 public class LoadManager implements Writable {
     private static final Logger LOG = LogManager.getLogger(LoadManager.class);
 
-    private Map<Long, LoadJob> idToLoadJob = Maps.newConcurrentMap();
+    private Map<Long, LoadJob> idToLoadJob = Collections.synchronizedMap(Maps.newLinkedHashMap());
     private Map<Long, Map<String, List<LoadJob>>> dbIdToLabelToLoadJobs = Maps.newConcurrentMap();
```

Comment:
Why replace concurrentMap with synchronizedMap ? what is the advantages and disadvantages?